.github/workflows/cd-deploy-nodes-gcp.yml

```diff
@@ -1,4 +1,4 @@
-name: CD
+name: Deploy Nodes to GCP
```

Comment:
Do we want to delay this large CI change until after the release?
I'll leave it to you and @upbqdn to decide, because he is the release manager.

Reply:
Same as in https://github.com/ZcashFoundation/zebra/pull/7682#discussion_r1357444420

Reply:
@upbqdn if by any means this delays you considerably for the release, just go ahead without these 2 PRs.

Reply:
This release will be big enough even without this change, so let's delay?